content/posts/june/June-Blog-Post.mdx

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
**question**: Was this file intended to be included, given that there was no previous equivalent?

Reply:
Yes! This is just a demo of the nested pages. I guess we could add better content for it but I think it at the very least demos it working